.github/workflows/eden.yml

```diff
@@ -15,141 +17,25 @@ on:  # yamllint disable-line rule:truthy
-        if: matrix.fs == 'ext4'
-        run: ./eden setup -v debug
-      - name: setup-zfs
-        if: matrix.fs == 'zfs'
-        run: |
-          ./eden config set default --key=eve.disks --value=4
-          ./eden config set default --key=eve.disk --value=4096
-          ./eden setup -v debug --grub-options='set_global dom0_extra_args "$dom0_extra_args eve_install_zfs_with_raid_level "'
-      - name: run
-        run: EDEN_TEST_STOP=n ./eden test ${{ github.workspace }}/eve/tests/eden/workflow -v debug
-      - name: Collect logs
-        if: ${{ always() }}
-        run: |
-          ./eden log --format json > trace.log || echo "no log"
-          ./eden info --format json > info.log || echo "no info"
-          ./eden metric --format json > metric.log || echo "no metric"
-          ./eden netstat --format json > netstat.log || echo "no netstat"
-          cp dist/default-eve.log console.log || echo "no device log"
-          docker logs eden_adam > adam.log 2>&1 || echo "no adam log"
-      - name: Log counting
-        if: ${{ always() }}
-        run: |
-          echo "::group::Total errors"
-          echo "$(jq '.severity' trace.log|grep err|wc -l)"
-          echo "::endgroup::"
-          echo "::group::Errors by source"
-          echo "errors by source: $(jq -s 'map(select(.severity|contains("err")))|group_by(.source)|map({"source": .[0].source, "total":length})|sort_by(.total)|reverse[]' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Error log content duplicates"
-          echo "$(jq -s 'map(select(.severity | contains("err")))|group_by(.content)|map(select(length>1))' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Error log function filename duplicates"
-          echo "$(jq -s 'map(select(.severity | contains("err")))|group_by(.filename)|map(select(length>10))|map({"source": .[0].source, "filename": .[0].filename, "function": .[0].function, "content": [.[].content], "total":length})|sort_by(.total)| reverse[]' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Segfaults"
-          echo "$(jq -s 'map(select(.content | contains("segfault at")))' trace.log)"|tee segfaults.log
-          [ "$(jq length segfaults.log)" -gt 0 ] && echo "::warning::segfaults found, you can see them in Log counting->Segfaults section"
-          echo "::endgroup::"
-      - name: Store raw test results
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v3
-        with:
-          name: eden-report-tpm-${{ matrix.tpm }}-${{ matrix.fs }}
-          path: |
-            ${{ github.workspace }}/trace.log
-            ${{ github.workspace }}/info.log
-            ${{ github.workspace }}/metric.log
-            ${{ github.workspace }}/netstat.log
-            ${{ github.workspace }}/console.log
-            ${{ github.workspace }}/adam.log
+  test_suite_pr:
+    if: github.event.review.state == 'approved'
+    uses: yash-zededa/eden/.github/workflows/test.yml@0.9.2
```

Comment:
@yash-zededa I think you might want to point to lf-edge repo

Reply:
Pointed to `lf-edge`